src/main/java/Entity/finalStatus.java

```diff
@@ -0,0 +1,8 @@
+package Entity;
+
+public enum finalStatus {
```

Comment:
Usually good to have enums with a starting capital letter.

`FinalStatus`

This allows us to differentiate easily what is a variable, and what is the enum.